lib/simplifyBasics/simplifyBasicsSearch.js

```diff
@@ -5,15 +5,18 @@
  * These are always the first simplifications that are attempted.
  */
 
-const clone = require('./clone');
-const MathChangeTypes = require('./MathChangeTypes');
-const Negative = require('./Negative');
-const NodeCreator = require('./NodeCreator');
-const NodeStatus = require('./NodeStatus');
-const NodeType = require('./NodeType');
-const PolynomialTermNode = require('./PolynomialTermNode');
-const PolynomialTermOperations = require('./PolynomialTermOperations');
-const TreeSearch = require('./TreeSearch');
+const clone = require('../util/clone');
+const MathChangeTypes = require('../MathChangeTypes');
+const Negative = require('../util/Negative');
+const NodeCreator = require('../util/NodeCreator');
+const NodeStatus = require('../util/NodeStatus');
+const NodeType = require('../util/NodeType');
+const TreeSearch = require('../util/TreeSearch');
+
+const reduceMultiplicationByZero = require('./reduceMultiplicationByZero');
+const reduceZeroDividedByAnything = require('./reduceZeroDividedByAnything');
+const rearrangeCoefficient = require('./rearrangeCoefficient');
+// TODO: move the rest of the functions into their own files too
```

Comment:
I might make this an issue, it's not super important but would feel cleaner I think